build.py

```diff
@@ -0,0 +1,42 @@
+import shutil
+from pathlib import Path
+
+from staticjinja import Site
+from yaml import Loader, load
+
+import sass
+
+
+def data():
```

Comment:
Function names must be made of verbs explaining what they do. Having only a noun suggests it's a constant/variable that doesn't do anything, as only verbs imply actions. This name is also very generic. It could've been `def things():` with the same amount of useful information in the name.
```suggestion
def read_page_definitions_from_disk():
```

Reply:
Thanks, valid point but I'll go with `load_page_data` which seems more descriptive. Love the level of detail in this review btw.